sdk/storage/azure_storage_blob/CHANGELOG.md

```diff
@@ -1,14 +1,21 @@
 # Release History
 
-## 0.2.0 (Unreleased)
+## 0.2.0 (2025-06-09)
 
 ### Features Added
 
+* Added navigation methods to access sub-clients from existing clients.
+* Added a new blob client type, `BlockBlobClient`.
+* Added support for `list_blobs`, `set_metadata` to `ContainerClient`.
+* Added support for `set_metadata`, `set_properties`, and `set_tier` to `BlobClient`.
+
 ### Breaking Changes
 
+* Moved `commit_block_list`, `get_block_list`, and `stage_block` from `BlobClient` to `BlockBlobClient`.
+
 ### Bugs Fixed
 
-### Other Changes
+* Fixed an issue where blob type would be present in RequestURI for certain APIs extraneously.
```

Comment:
nit: `RequestURI` seems like the name of the field in the recording? Either way, not particularly helpful for customers. Maybe just say "request URL"?

Maybe we should also mention that it is a query parameter. "Fixed an issue where the blob type string would appear as query parameter in the request URL for certain APIs extraneously."

Reply:
Thanks, taking that as-is 😄